lib/Symbols.js

```diff
@@ -71,7 +101,17 @@ function isSymbolTerm(node, symbolName) {
       return true;
     }
   }
+  else if (hasDenominatorSymbol(node, symbolName)) {
```

Comment:
Jumping back to this because it should probably be discussed (and have code/comments updated appropriately).  Prior to this change, `isSymbolTerm` specifically was checking only for **polynomial** symbol terms.  So a term like `1/x` would not be considered a symbol term because it's not a polynomial term.  This change makes it so that terms with a symbol in the denominator are considered symbol terms as well.

In turn, this affects `getLastNonSymbolTerm` and `getLastSymbolTerm` which both use `isSymbolTerm`.  I'm not sure if this change is going against the intentions of the Symbols object.  If it is, then I'll need to find some other way to get a symbol term in a denominator